src/edits.ts

```diff
@@ -398,3 +408,10 @@ async function uploadReleaseFiles(appEditId: string, options: EditOptions, relea
 
     return versionCodes
 }
+
+function inferInternalSharingDownloadUrl(
+  applicationId: string,
+  versionCode: number
+) {
+  return `https://play.google.com/apps/test/${applicationId}/${versionCode}`;
```

Comment:
Has this been tested beforehand? I want to make sure there's no special config needed for it, or if there is that it's documented

Reply:
I have tested it myself. Comparing to deploying through the `internalsharing ` track, this does require [whitelisting users](https://support.google.com/googleplay/android-developer/answer/9844279?_ga=2.184121315.453176378.1672850123-1144302356.1658411817#:~:text=Tip%3A%20You%20can%20select%20Manage%20access%20to%20visit%20the%20Internal%20app%20sharing%20page%20and%20quickly%20share%20app%20bundle%20and%20APK%20links%20with%20your%20team.%20To%20learn%20more%2C%20go%20to%20Share%20app%20bundles%20and%20APKs%20internally.). Otherwise, it should work the same.